src/strings/PowerFxResources.en-US.resx

```diff
@@ -4115,6 +4115,10 @@
     <value>Named formula must be an expression.</value>
     <comment>This error message shows up when Named formula is not an expression. For example, a = ;</comment>
   </data>
+  <data name="ErrNamedType_MissingTypeLiteral" xml:space="preserve">
+    <value>Named type must be a TypeLiteral.</value>
+    <comment>This error message shows up when Named type is not a TypeLiteral. For example, T := ; T2 := 5;</comment>
```

Comment:
>TypeLiteral [](http://example.com/codeflow?start=66&length=11)

Can you also explain what a type literal is (i.e., give _valid_ definitions of a type)? This way the translators will have a better understanding of this term.